src/libostree/ostree-bootloader-grub2.c

```diff
@@ -246,6 +247,19 @@ _ostree_bootloader_grub2_generate_config (OstreeSysroot                 *sysroot
           g_string_append_c (output, '\n');
         }
 
+      devicetree = ostree_bootconfig_parser_get (config, "devicetree");
+      if (devicetree)
+        {
+          g_string_append (output, "devicetree");
+          if (is_efi)
+            g_string_append (output, GRUB2_EFI_SUFFIX);
+          else
+            g_string_append (output, GRUB2_SUFFIX);
```

Comment:
I don't think this `if (is_efi)` part is necessary right?  That turns into e.g. `16` or `efi` sometimes for e.g. `linux16` or `linuxefi`, but is it really e.g. `devicetreeefi`?  (Are these devices EFI?)

https://www.gnu.org/software/grub/manual/grub/html_node/devicetree.html

Seems to imply it's always `devicetree`.

Reply:
You are right, it should always be devicetree.

Reply:
Let me push a fixup for this.